test/abnormal_states_recovery_test.rb

```diff
@@ -198,7 +198,7 @@ def with_invalidation_while_executing(finish)
           plan = client_world.persistence.load_execution_plan(triggered.id)
           plan.state.must_equal :paused
           expected_history = [['start execution', executor_world.id],
-                              ['finish execution', executor_world.id]]
+                              ['pause execution', executor_world.id]]
```

Comment:
Added a new kind of execution history for paused event so that it's more descriptive on what happened.